index.html

```diff
@@ -267,7 +267,7 @@ <h2>Skills</h2>
         <div class="contact-div">
           <h1>Contact me</h1>
           <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
-          <form class="contact-form" action="" method="post">
+          <form class="contact-form" action="https://formspree.io/f/{form_id}" method="post">
```

Comment:
- [ ] Well done using the post method on the form as suggested by the previous reviewer , but you are required to sign up on the [formspree service](https://formspree.io/html) and integrate your form using the form_id, Kindly fix it , When I submit the form it show me the following message.

![Capture](https://user-images.githubusercontent.com/93598189/205949646-9e30a750-b7fb-40a3-a184-c2277a316106.PNG)